app/src/main/java/com/xinto/opencord/ui/viewmodel/ChatViewModel.kt

```diff
@@ -42,6 +43,8 @@ class ChatViewModel(
         private set
     var sendEnabled by mutableStateOf(true)
         private set
+    var currentUserId by mutableStateOf<Long?>(null)
```

Comment:
Could be 0 instead of null

Reply:
Why though, its set from ReadyEvent so it'll always be missing or valid. null exists to represent absense of data

Reply:
I guess so if ready event doesn't fire for whatever reason

Reply:
Therefore null?

Reply:
Okay yeah sure